crates/ln-dlc-node/src/config.rs

```diff
@@ -53,6 +53,7 @@ pub fn app_config() -> UserConfig {
         },
         channel_config: ChannelConfig {
             cltv_expiry_delta: MIN_CLTV_EXPIRY_DELTA,
+            accept_underpaying_htlcs: true,
```

Comment:
I assume you didn't want this change in the commit: 4af7474602044b343783d7dd49689f0ad722085e but in the next

Reply:
🔧 Documenting this config was always really helpful.

```suggestion
            // Allows the coordinator to charge us a channel-opening fee after intercepting the app's funding HTLC.
            accept_underpaying_htlcs: true,
```